README.md

```diff
@@ -15,9 +15,9 @@ worldengine world -s 1 -n seed1
 
 Worlds are generated using plate simulations, erosion, rain shadows, Holdridge life zones model and plenty of other phenomenons.
 
-Once a world it can be used for simulation civs evolution (see project [civs](https://github.com/ftomassetti/civs)).
+A finished world can be used for simulating the evolution of a civilization (see project [civs](https://github.com/ftomassetti/civs)).
```

Comment:
"A finished world" is fine. In fact it's much better than his or read
before.
On Nov 17, 2015 14:41, "Federico Tomassetti" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Mindwerks/worldengine/pull/202#discussion_r45133722>:
>
> > @@ -15,9 +15,9 @@ worldengine world -s 1 -n seed1
> >
> >  Worlds are generated using plate simulations, erosion, rain shadows, Holdridge life zones model and plenty of other phenomenons.
> >
> > -Once a world it can be used for simulation civs evolution (see project [civs](https://github.com/ftomassetti/civs)).
> > +A finished world can be used for simulating the evolution of a civilization (see project [civs](https://github.com/ftomassetti/civs)).
>
> "A finished world" sounds weird to me
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mindwerks/worldengine/pull/202/files#r45133722>.
>